### DISCUSSION TITLE
Failed stack update, now DNS issues? Any pointers?

### DISCUSSION THREAD

**Discussion by gbayles** [mozilla/hubs#3277] [Hubs Cloud Questions] [answered]
Very new to Cloud Hubs, but after updating a working Hubs stack to scale it up, it encountered errors and couldn't roll back successfully. Took all servers offline temporarily to try to refresh, and they seem to be back up and functioning normally now, but now I'm getting a DNS error, and the health check on Route 53 is coming back as "Unhealthy." Any pointers on how to fix?

Both domains are hosted through Route 53 and were functioning as expected before the failed update. DNS error message below:

This site can’t be reached 
virtualholidaymarket.live’s server IP address could not be found.
DNS_PROBE_FINISHED_NXDOMAIN

**Comment by robinkwilson** [comment, marked as answer]
No worries! Let's see what we can do. A few questions for you:

Can you take a few screenshots where you're seeing this error?
Can you explain what steps you took here: "took all servers offline temporarily to try to refresh"?
Likewise, can you explain the steps you took here too so I'm totally clear: "and they seem to be back up and functioning normally now"?

Thanks!

**Reply by gbayles** [reply]
Thanks for the reply!

Attaching a few screenshots below.

Basic steps of what happened below:
1. Was working inside of our already set-up Hub Cloud environment.
2. Used CloudFormation to update the stack to larger sized servers, as a test. 
3. Update failed, and the front-end environment (Hubs Admin Console, rooms, etc) became inaccessible.
4. Rollback failed.
5. Tried a bunch of things like ContinueRollback with it failing each time, giving a message that it wasn't receiving signals from the servers on the stack update Wait Command.
6. Finally got it to say Rollback Complete.
7. Front-end environment still wasn't available (Hubs Admin Console, rooms, etc)
8. Updated CloudFormation stack to go offline completely.
9. Updated CloudFormation stack to go online again.
10. Stack is up and presumably functioning, but front end gives a DNS error.
11. Tried deleting the CNAMES inside of Route 53, like it suggests [here](https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html).
12. Updated CloudFormation stack to reassign CNAMES, but that didn't seem to do anything. CNAME records didn't repopulate.

![DNS error](https://user-images.githubusercontent.com/9869708/98282139-fe5da600-1f5a-11eb-9b94-8afd6ae3595a.png)
![Update Failed](https://user-images.githubusercontent.com/9869708/98282144-fef63c80-1f5a-11eb-8dba-15cbc5a2450d.png)
![Stack Updated](https://user-images.githubusercontent.com/9869708/98282145-ff8ed300-1f5a-11eb-8b67-51efb25833f4.png)

**Reply by gbayles** [reply]
I guess to overcommunicate, if the easiest thing to do would be to just start over, I'm not opposed to that. But we have a good amount of work sunk into the scenes and stuff, and I'm not familiar enough with the system to know what gets kept and what gets deleted if I were to delete the stack and deploy from scratch.

**Reply by robinkwilson** [reply]
Thank you for the detailed steps!!

I'll need to change the documentation ASAP! The CNAMES are required when you have a running stack -- that only applies when your stack is not being build properly.
Unfortunately, I made the stack work after deleting a CNAME that's mapped to a running resource _-- happened to my stack a bit ago._

To save your scenes, you'll need to do a [backup and restore](https://hubs.mozilla.com/docs/hubs-cloud-aws-backup-and-restore.html). Not ideal. Let me know how it goes!
